package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "weakauras-companion",
-  "version": "3.0.0-beta1",
+  "version": "3.0.0-beta2",
```

Comment:
I'd like to keep this change in a separate commit i'll do myself when i do pre-release, i have to prepare the draft before changing this.